src/modules/leaderboard.js

```diff
@@ -0,0 +1,47 @@
+class Leaderboard {
+  constructor() {
+    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+  }
+
+  async startGame(gameName) {
```

Comment:
* According to the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20arrow%20functions%20instead%20of%20the%20function%20keyword.) you are required to Use arrow functions instead of the function keyword, Kindly use arrow functions.